### PR TITLE
Mostly bug fixing for brsa

### PR DESCRIPTION
**Pull request by lcnature**
And some typo corrections.
The fitV and fitU steps in the _fit_diagV_noGP and _fit_diagV_GP functions are swapped when debugging. But I think the order does not matter so I did not swap them back.
Update example notebook to mention that z-scoring data is necessary.

### CODE REVIEW

File: brainiak/reprsimil/brsa.py [hunk 1]
```diff
@@ -1291,16 +1297,22 @@ def _loglike_AR1_diagV_fitU(self, param, XTX, XTDX, XTFX, YTY_diag,
...
         # Only starting from this point, SNR2 is involved
-        LL, LAMBDA_i, LAMBDA, YTAcorrXL_LAMBDA, sigma2 \
-            = self._calc_LL(rho1, LTXTAcorrXL, LTXTAcorrY, YTAcorrY, X0TAX0,
-                            SNR2, n_V, n_T, n_run, rank, n_base)
+
+        try:
+            LL, LAMBDA_i, LAMBDA, YTAcorrXL_LAMBDA, sigma2 \
+                = self._calc_LL(rho1, LTXTAcorrXL, LTXTAcorrY, YTAcorrY,
+                                X0TAX0, SNR2, n_V, n_T, n_run, rank, n_base)
+        except:
```
Comment (mihaic) [inline brainiak/reprsimil/brsa.py]: You must specify a particular exception and you must add that exception to the docstring of the function in a section called "Raises".

File: brainiak/reprsimil/brsa.py [hunk 2]
```diff
@@ -1291,16 +1297,22 @@ def _loglike_AR1_diagV_fitU(self, param, XTX, XTDX, XTFX, YTY_diag,
...
-            logger.debug(YTAcorrY)
-            logger.debug(LTXTAcorrY)
-            logger.debug(YTAcorrXL_LAMBDA)
-            logger.debug(SNR2)
+            logger.debug('LL: {}'.format(LL))
+            logger.debug('sigma2: {}'.format(sigma2))
+            logger.debug('YTAcorrY: {}'.format(YTAcorrY))
+            logger.debug('LTXTAcorrY: {}'.format(LTXTAcorrY))
+            logger.debug('YTAcorrXL_LAMBDA: {}'.format(YTAcorrXL_LAMBDA))
+            logger.debug('SNR2: {}'.format(SNR2))
```
Comment (mihaic) [inline brainiak/reprsimil/brsa.py]: Have you thought if `warnings.warn()` or `logger.warning` are appropriate here?

File: examples/reprsimil/brsa_representational_similarity_estimate_example.ipynb [hunk 3]
```diff
@@ -7,7 +7,7 @@
    },
    "source": [
     "# This demo shows how to use the Bayesian Representational Similarity Analysis method in brainiak with a simulated dataset.\n",
-    "Questions can be directed to mcai [ at ] princeton [ dot ] edu"
+    "*Feedbacks and questions are welcome. Please direct them to mcai [ at ] princeton [ dot ] edu"
```
Comment (mihaic) [inline examples/reprsimil/brsa_representational_similarity_estimate_example.ipynb]: Not really what the change is about, but do you mind removing this line? We are already instructing users to open GitHub issues for BrainIAK or contact us on Gitter.

Comment (lcnature) [inline examples/reprsimil/brsa_representational_similarity_estimate_example.ipynb]: OK!

File: brainiak/reprsimil/brsa.py (hunk 1 above)
Comment (lcnature) [inline brainiak/reprsimil/brsa.py]: OK I removed the try/except

**Pull request merged**